specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-02/iotdps.json

```diff
@@ -1915,6 +1915,10 @@
         "enableDataResidency": {
           "type": "boolean",
           "description": "Optional.\nIndicates if the DPS instance has Data Residency enabled, removing the cross geo-pair disaster recovery."
+        },
+        "portalOperationsHostName": {
+          "type": "string",
+          "description": "Portal endpoint to enable CORS for this provisioning service."
```

Comment:
>Portal endpoint to enable CORS for this provisioning service. [](http://example.com/codeflow?start=26&length=61)

Just curious, if you are looking for Azure Portal endpoint here, using which the client made this call? FYI: standard client headers which are passed to RPs by ARM, including "referrer" header: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#proxy-request-header-modifications](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#proxy-request-header-modifications).